engine/engine/src/test/label/label.script

```diff
@@ -13,38 +13,44 @@
 -- specific language governing permissions and limitations under the License.
 
 
+local function assert_near(exp, act, eps)
+	assert(math.abs(exp - act) < eps, string.format("expected %f but was %f, differing by %f which exceeds %f", exp, act, math.abs(exp - act), eps))
+end
+
+local epsilon = 0.000001
+
 local function compare_fp(lhs, rhs)
-    local epsilon = 1e-10
-    local diff = math.abs(lhs - rhs)
-    return epsilon > diff
+	local epsilon = 1e-10
+	local diff = math.abs(lhs - rhs)
+	return epsilon > diff
 end
 
 local function test_vector4(name, expected)
 
-    local v = go.get("#label", name)
-    local vx = go.get("#label", name .. ".x")
-    local vy = go.get("#label", name .. ".y")
-    local vz = go.get("#label", name .. ".z")
-    local vw = go.get("#label", name .. ".w")
-
-    assert(expected == v)
-    assert(expected.x == vx)
-    assert(expected.y == vy)
-    assert(expected.z == vz)
-    assert(expected.w == vw)
+	local v = go.get("#label", name)
```

Comment:
Replaced spaces with tabs?
Makes it harder to detect/read the actual changes.

Reply:
yes, I just made it consistent because most of the file uses tabs, and only a bit of code spaces.

Reply:
well, in that case it should be the other way around (we use spaces in our code style) :)